source/code/mof/docker.mof

```diff
@@ -157,4 +157,36 @@ class Container_ContainerInventory : CIM_ManagedElement
 
 	[ Description ( "JSON of container links" ) ]
 	String Links;
-};
\ No newline at end of file
+};
+
+// Container_ContainerLog
+// -------------------------------------------------------------------
+[   Version ( "1.0.0" ),
+	Description ( "Docker Container Logs Information" )
+]
+class Container_ContainerLog : CIM_ManagedElement
```

Comment:
Generally, new classes have to be "blessed" from PM to make sure they abide by the CIM guidelines. Was this done? If @agup006 isn't comfortable with that, you might want to reach out to @KrisBash to be sure that this is "right".